masterlist.yaml

```
@@ -3467,6 +3467,24 @@ plugins:
       - crc: 0x796B5447
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: 'HPP - (Vaermina''s Torpor|FirewoodSnow|FirewoodSnow Campfire).*\.esp'
```

You only need the quantifier `*` if you need capture any plugins that start with 
`HPP - (Vaermina''s Torpor|FirewoodSnow|FirewoodSnow Campfire)`

[Regex Cheat Sheet](https://www.rexegg.com/regex-quickstart.html)

```suggestion
  - name: 'HPP - (Vaermina''s Torpor|FirewoodSnow|FirewoodSnow Campfire)\.esp'
```